os_migrate/playbooks/export_user_keypairs.yml

```diff
@@ -0,0 +1,3 @@
+- hosts: migrator
+  roles:
+    - os_migrate.os_migrate.export_user_keypairs
```

Comment:
I think i'll still rename the playbook and role to `export_users_keypairs`, meaning "export users' keypairs". I'd like it to be more obvious from the playbook name that when executed, it will do an `O(n^2)` type of operation.